ecommerce/docs/api/data-models.md

```
@@ -0,0 +1,1659 @@
+    20,  -- 초기 재고 20개
+    0    -- 버전 0
+FROM box_combination;
+```
+
+---
+
+## 7. 성능 최적화 고려사항
+
+### 7.1 읽기 최적화
+
+#### 상품 정보 캐싱 (Redis)
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)